### ISSUE TITLE
Feature Request: T2I Adapter Support

### ISSUE THREAD

**Issue by JustCann** [guoyww/AnimateDiff#279]
Hello,

I may be doing this incorrectly, but when I attempt to use T2I Adapters in CN the extension creates an animated gif but keeps the model frozen in place as if I was doing img2img. Is using T2I Adapters supported or am I doing something incorrectly? 

Thanks!!

**Comment by vicitooo** [comment]
Lower your strength and percents. It is almost like tile. It will fix it in place with minimal movement if your CN is too strong.

**Comment by vicitooo** [comment]
No, you can run as many CN as you want. The more CN and the higher the weights, the more stable the image is. It is a fine balance between stability and motion.

**Comment by JustCann** [comment]
I did what you recommended, this time I only used t2IAdapter depth and the same issue still occurred, no movement as compared to other CN.

**Comment by JustCann** [comment]
I am just coming to the conclusion that T2I is just not possible with AnimateDiff. If I wanted a still image, I would make one, the caveat is the generation time. :(
